src/crimson/osd/main_config_bootstrap_helpers.cc

```diff
@@ -84,6 +84,58 @@ seastar::future<> populate_config_from_mon()
   });
 }
 
+enum class OptionValue {
+  Uint,
+  Double
+};
+
+// Struct to define a Seastar configuration option
+struct SeastarOption {
+  std::string option_name;  // Command-line option name
+  std::string config_key;   // Configuration key
+  OptionValue value_type;   // Type of configuration value
+};
+
+// Define a list of Seastar options
+const std::vector<SeastarOption> seastar_options = {
+  {"--task-quota-ms", "crimson_reactor_task_quota_ms", OptionValue::Double},
+  {"--io-latency-goal-ms", "crimson_reactor_io_latency_goal_ms", OptionValue::Double},
+  {"--idle-poll-time-us", "crimson_reactor_idle_poll_time_us", OptionValue::Uint}
+};
+
+// Template function to fetch values from configuration
+template <typename T>
+std::optional<T> get_conf_value(const std::string& key) {
+  try {
+    return crimson::common::get_conf<T>(key);
+  } catch (const std::exception& e) {
```

Comment:
Why are we expecting `get_conf` to throw?

Reply:
Not required, remove it.